scripts/triggerRoleUpdate.ts

```diff
@@ -19,27 +19,20 @@ const db = admin.firestore();
 //cole id 619663424812613662
 // const discordID = "619663424812613662";
 
-const discordID = "877111583879344158";
+// const discordID = "877111583879344158";
+const address = "oijoij";
 
 const triggerRoleUpdate = async () => {
-  const userDoc = await db
-    .collection("users")
-
-    .doc(discordID)
-    .get();
-
-  console.log(userDoc);
-  console.log(userDoc.exists);
-  console.log(userDoc.data());
-  //   const newUser: User = {
-  //     wallet: "terra1zu5vxgnq48rz3ypjwlsc9luj5zua9kmsrt7r4m",
-  //   };
-  const newUser: User = { wallet: (userDoc.data() as User).wallet };
-
-  console.log("Deleting user");
-  await db.collection("users").doc(discordID).delete();
-  console.log("Setting user", newUser);
-  await db.collection("users").doc(discordID).set(newUser);
+  const discordId = await passportApi.getDiscordIdByWallet(address);
+
+  // unlink wallet
+
+  console.log("Unlinking wallet");
+  await passportApi.unlinkAddressFromDiscord(address);
+
+  // relink wallet
+  console.log("Relinking wallet");
+  await passportApi.linkAddressToDiscordId(address, discordId);
```

Comment:
With the bot originally - I assume relinking it triggered the websocket event and let the bot know to update the roles?

Reply:
Yes exactly!

Reply:
How about now though? w/o socket, does this relinking actually trigger the bot to update roles?